dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -327,32 +378,42 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent
                         .setPageMode(PageMode.LIVE)
                         .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ESP = "+html , html.contains("content3content2Spa"));
+        Assert.assertTrue("ESP = "+html , html.contains("content3Spacontent2Spa"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "content3Spacontent2Spa" appears 4 times in this file; the first occurrence is on line 381](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)